drivers/char/rust_example.rs

```diff
@@ -74,6 +79,24 @@ impl KernelModule for RustExample {
             println!("  my_usize:   {}", my_usize.read(&lock));
         }
 
+        // Test mutexes.
+        {
+            // SAFETY: `init` is called below.
+            let data = Pin::from(Box::try_new(unsafe { Mutex::new(0) })?);
+            mutex_init!(data.as_ref(), "RustExample::init::data1");
+            *data.lock() = 10;
+            println!("Value: {}", *data.lock());
+        }
+
+        // Test spinlocks.
+        {
+            // SAFETY: `init` is called below.
+            let data = Pin::from(Box::try_new(unsafe { SpinLock::new(0) })?);
+            spinlock_init!(data.as_ref(), "RustExample::init::data2");
+            *data.lock() = 10;
+            println!("Value: {}", *data.lock());
+        }
+
```

Comment:
Missing the condvar test.

Reply:
I've added some code there for very basic testing.